test/e2e/service_to_service_test.go

```diff
@@ -219,7 +229,11 @@ func TestServiceToServiceCallViaActivator(t *testing.T) {
 	defer test.TearDown(clients, testNames)
 
 	resources, err := v1a1test.CreateRunLatestServiceReady(t, clients, &testNames,
-		&v1a1test.Options{},
+		&v1a1test.Options{
+			RevisionTemplateAnnotations: map[string]string{
+				"sidecar.istio.io/inject": strconv.FormatBool(injectB),
+			},
```

Comment:
Can you move this into the `WithConfigAnnotations` helper below? I think it's kinda redundant otherwise.

Reply:
Updated. I missed the fact that WithConfigAnnotations could do the same thing.
Thanks.